video/out/drm_atomic.c

```diff
@@ -263,15 +263,16 @@ struct drm_atomic_context *drm_atomic_create_context(struct mp_log *log, int fd,
             mp_verbose(log, "Using default plane %d for video\n", overlay_id);
             ctx->video_plane = drm_object_create(log, ctx->fd, overlay_id, DRM_MODE_OBJECT_PLANE);
         } else {
-            mp_err(log, "Failed to find video plane with id=%d\n", video_plane_id);
-            goto fail;
+            mp_verbose(log, "Failed to find video plane with id=%d. drmprime-drm hwdec interop will not work\n", video_plane_id);
         }
     } else {
         mp_verbose(log, "Found video plane with ID %d\n", ctx->video_plane->id);
     }
 
-    mp_verbose(log, "Found Video plane with ID %d, OSD with ID %d\n",
-               ctx->video_plane->id, ctx->osd_plane->id);
+    if (ctx->video_plane) {
+        mp_verbose(log, "Found Video plane with ID %d, OSD with ID %d\n",
```

Comment:
I would vote for something like this if you want to keep the message (this would let you remove the individual `else`s:
```c
mp_verbose(log, "Planes found: video: %d, OSD: %d (-1 = \"not found\")\n",
           ctx->video_plane ? ctx->video_plane->id : -1,
           ctx->osd_plane ? ctx->osd_plane->id : -1);
```

Otherwise since both the video and OSD planes' else things log what they found, that might also be enough. I'd prefer to have one status output at the end of the process, of course.

Reply:
This is good, except that in my in-going PR I use -1 to mean "use any primary plane" and -2 to mean "use any overlay plane" thus this usage of -1 = not found would collide with that.